src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IDelegateCreationExpression.vb

```diff
@@ -1785,6 +1785,190 @@ BC30002: Type 'NonExistant' is not defined.
             VerifyOperationTreeAndDiagnosticsForTest(Of ObjectCreationExpressionSyntax)(source, expectedOperationTree, expectedDiagnostics)
         End Sub
 
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub DelegateCreationExpression_ParenthesizedLambda_CType()
+            Dim source = <![CDATA[
+Imports System
+
+Public Class C
+    Public Sub M1()
+        CType(((Sub() Console.WriteLine())), Action)'BIND:"CType(((Sub() Console.WriteLine())), Action)"
```

Comment:
>((Sub() Console.WriteLine())) [](start = 14, length = 29)

Do we have a test for a scenario when this parenthesized expression is implicitly converted?

Reply:
Added one.

---
In reply to: [152436335](https://github.com/dotnet/roslyn/pull/23315#discussion_r152436335) [](ancestors = 152436335)